examples/fpga_flow/techlib/example_arch_bram_memory_map.txt

```diff
@@ -0,0 +1,12 @@
+ram block $_tech_bram_ {
+        abits 13;
+	byte 8;
+        widths 1 2 4 8 per_port;
+        cost 3;
+        init none;
+        port srsw "A" "B" {
+                clock posedge;
+                rden;
+        }
+
+}
```

Comment:
could you add a set of `# comments` to describe the type of memory getting mapped here